tests/unit/color/test_colorscale.py

```diff
@@ -56,7 +57,10 @@ def test_validate_and_coerce_colorscale_fails(
 
 
 def test_list_all_colorscale_names() -> None:
-    all_colorscale_names = list_all_colorscale_names()
+    with pytest.warns(
+        DeprecationWarning, match=re.escape("list_all_colorscale_names() is deprecated")
```

Comment:
**Another small tip:** using `re.escape("my(special)string.")` can be simpler and more readable than `r"my\(special\)string\."`

Reply:
It makes sense. Great tip again :smile: